electronicparsers/wien2k/parser.py

```diff
@@ -1053,7 +1097,40 @@ def parse_method(self):
             sec_method.k_mesh = sec_k_mesh
             sec_k_mesh.points = kpoints[0]
             sec_k_mesh.weights = kpoints[1]
-
+        if self.inc_parser.mainfile:
+            self.inc_parser.parse()
+            atoms = self.inc_parser.get('one_atom')
+            for atom_index, atom in enumerate(atoms):
+                kappas = atom.get('kappas')
+                n_quantum_numbers = atom.get('n_quantum_numbers')
+                occupancies = atom.get('occupancies')
+                number_of_orbitals = len(kappas)
+                for orbital in range(number_of_orbitals):
+                    max_occupancy = abs(kappas[orbital] * 2)
+                    occupancy = int(occupancies[orbital])
+                    if occupancy < max_occupancy:
+                        if kappas[orbital] >= 0:
+                            spin_quantum_number = -1
+                        else:
+                            spin_quantum_number = 1
+                            j_quantum_number = (
+                                -kappas[orbital] / spin_quantum_number - 1 / 2
+                            )
+                        l_quantum_number = j_quantum_number - (spin_quantum_number / 2)
```

Comment:
Nice :smiley:, I would have probably just hardcoded the table to a dict, but this is sure more concise, and if we add link to the table, it should be fine... Just make sure to comment that this calculations are based on the table.